packages/search/src/util/pagination/content-paginator.ts

```diff
@@ -0,0 +1,48 @@
+import { encode } from "base-64";
+import { IPageResponse } from "./paginator";
+
+interface IContentPageStart {
+  hub: number;
+  ago: number;
+}
+
+interface IContentPage {
+  hubAdded: number;
+  agoAdded: number;
```

Comment:
It's not clear to me what these represent. Number of pages to add to the current position for each? Or are they records? Generally it would be good to document w/ code comments (which will show up in docs) all the members of each interface.

Reply:
Good call. I changed these to `hubRecordsAdded` and `agoRecordsAdded`, respectively. I also changed the other two props to `hubRecordsTotal` and `agoRecordsTotal` and added documentation to the interfaces created as part of this PR.